icubam/backoffice/templates/operational-dashboard.html

```diff
@@ -1,113 +1,95 @@
-{% extends "base.html" %}
+{% extends "operational-dashboard-base.html" %}
 
-{% block links %}
-<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
-<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
-<link rel="stylesheet" type="text/css" href="static/dist/css/dashboard.css">
-
-{% end %}
-
-{% block content %}
-<div class="container operational-dash">
-    <div class="row text-center">
-    <h1 class='text-center'>{{ _("Operational dashboard") }}</h1>
-    </div>
+<!-- Dashboard filter navbar -->
+{% block dashboard_filter %}
```

Comment:
This diff is a bit hard to review, but essentially I split `operational-dashboard.html` into two templates: one base template with the general page layout and this file with the HTML code of each block.

So far the functionality is not changed, but this will help putting these different blocks into tabs.